doc/tutorials/storage-plugins.md

```diff
@@ -207,10 +207,95 @@ kdb rm -r user:/tests/storage
 sudo kdb umount user:/tests/storage
 ```
 
-.
+## Storing Comments
+
+Most markup languages provide the possibility of adding comments. Elektra can store those comments in its meta-information as well. This can be achieved by setting the meta Key `comment` for the respective configuration Key. The `hosts` plugin currently stores the comments of the file in the respective Elektra configuration Key:
```

Comment:
```suggestion
Most markup languages provide the possibility of adding comments.
Elektra can store those comments in its meta-data as well.
This can be achieved by setting the meta Keys `comment/#` for
the respective configuration Key. Also the `hosts` plugin stores
the comments of the file in the respective Elektra configuration Key:
```